lib/item_repository.rb

```diff
@@ -72,9 +66,14 @@ def create(attributes)
   end
 
   def update(id, attributes)
```

Comment:
Consider changing this method to not require the entire attribute hash, bit instead the specific changes that need to be made.

Reply:
For the create method it should automatically assign a new (next highest id), so that a user does not have to manually look it up.